src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestBindCallGen.generated.txt

```diff
@@ -0,0 +1,122 @@
+        temp4 ??= new System.Collections.Generic.Dictionary<string, Program.MyClass2>();
+        BindCore(configuration.GetSection("MyComplexDictionary"), ref temp4);
+        obj.MyComplexDictionary = temp4;
+
+    }
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref System.Collections.Generic.List<int> obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+        int element;
+        foreach (Microsoft.Extensions.Configuration.IConfigurationSection section in configuration.GetChildren())
+        {
+            if (section.Value is string stringValue5)
+            {
+                element = int.Parse(stringValue5);
+                obj.Add(element);
+            }
+        }
+    }
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref System.Collections.Generic.Dictionary<string, string> obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+        string key;
+        foreach (Microsoft.Extensions.Configuration.IConfigurationSection section in configuration.GetChildren())
+        {
+            if (section.Key is string stringValue6)
+            {
+                key = stringValue6;
+                string element;
+                if (section.Value is string stringValue7)
+                {
+                    element = stringValue7;
+                    obj[key] = element;
+                }
+            }
+        }
+    }
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref System.Collections.Generic.Dictionary<string, Program.MyClass2> obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+        string key;
+        foreach (Microsoft.Extensions.Configuration.IConfigurationSection section in configuration.GetChildren())
+        {
+            if (section.Key is string stringValue8)
+            {
+                key = stringValue8;
+                if (obj.TryGetValue(key, out Program.MyClass2? element) && element is not null)
+                {
+                    BindCore(section, ref element);
+                    obj[key] = element;
+                }
+                else
+                {
+                    element = new Program.MyClass2();
+                    BindCore(section, ref element);
+                    obj[key] = element;
+                }
+            }
+        }
+    }
+
+    private static void BindCore(global::Microsoft.Extensions.Configuration.IConfiguration configuration, ref Program.MyClass2 obj)
+    {
+        if (obj is null)
+        {
+            throw new global::System.ArgumentNullException(nameof(obj));
+        }
+
+    }
```

Comment:
What's the point of this method?

Reply:
This is because the type has no bindable members. I'll make changes to avoid generation in those cases.